unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -24,11 +24,12 @@ const ENDING_SPACE_PERIOD_REGEX = /[ .]+$/;
 const STARTING_SPACE_PERIOD_REGEX = /^[ .]+/;
 
 class ServiceHandler {
-  async fetchFromService(url, options) {
+  async fetchFromService(url, options, canRetry = true) {
     try {
       const response = await fetch(url, options);
       const contentLength = response.headers.get('Content-Length');
       if (response.status === 202) return { status: 202, headers: response.headers };
+      else if(canRetry && response.status >= 500 && response.status < 600) {  return this.fetchFromService(url, options, false); }
```

Comment:
should we add a delay before retry for server error?
```suggestion
  if(canRetry && response.status >= 500 && response.status < 600) {
    await new Promise(resolve => setTimeout(resolve, 1000));
    return this.fetchFromService(url, options, false);
 }
```